### ISSUE TITLE
[Feature request] Menu bar background

### ISSUE THREAD

**Issue by flanter21** [AerialScreensaver/Aerial#76]
Can we have an option so that the background will extend behind the menu bar on the top of the screen when using the video background feature? I'm using a notched laptop if that makes a difference. I've put screenshots below to better explain what I mean.

![image](https://github.com/AerialScreensaver/AerialCompanion/assets/139064898/2d99b2ef-7bfd-4194-88df-f1c933a6b62a)
![image](https://github.com/AerialScreensaver/AerialCompanion/assets/139064898/b5f9c5d0-6fba-4533-b882-6e956a665a6b)

**Comment by glouel** [comment]
Hey @flanter21 

It's on my todo list but I need to manipulate the user's desktop background to make it work. It's a bit messy so I want to be sure I get it 100% right.

**Comment by flanter21** [comment]
Oh awesome. Thanks for such a quick reply and thank you for making such a useful app freely available. It's so refreshing to use when studying flashcards.

**Comment by chriswep** [comment]
> It's on my todo list but I need to manipulate the user's desktop background to make it work. It's a bit messy so I want to be sure I get it 100% right.

i suppose it would kind of work, if you set the desktop background to a static image of the first frame of the current video?

@flanter21 as it is now, i found it to look best on a macbook with notch when you set the desktop background to just black color, which makes the menubar just black (and hides the notch). however a menubar in sync with the current aerial desktop video would look even better of course.

**Comment by glouel** [comment]
> suppose it would kind of work, if you set the desktop background to a static image of the first frame of the current video?

Yes that's the basic idea, the hard part is handling it correctly : 
- restoring the macOS wallpaper on exit/request (which is now way more complicated since Ventura)
- handling multiple monitors 
- handling multiple spaces

That's also how I plan to tackle the login screen issue, this time by setting the last played frame as the desktop. 

Still ongoing.

**Comment by chriswep** [comment]
> Yes that's the basic idea, the hard part is handling it correctly :

I understand, all those edge cases make it complicated. However for my use case just setting and forgetting the wallpaper to every desktop would get me 95% of the way already. So maybe it could be an experimental/advanced checkbox to do that until its implemented properly..

**Comment by glouel** [comment]
The menu bar background has disappeared in recent macOS versions, so closing this. Video plays fullscreen under the menu bar now. 

A solid background option may be something I add back to companion, I'm not a fan of transparent menus (and Tahoe design in general, but I'm not alone here). May revisit this soon.
